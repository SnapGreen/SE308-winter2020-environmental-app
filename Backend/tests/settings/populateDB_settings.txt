settings check:
	SETTINGS: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/scripts/settings.txt
	SERVER_POPULATED: false
	LASTUPLOAD: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/data/FDA/branded_food_0451.json
	USAGE:
		Usage: ./populateDB.sh [OPTION] (use option -h for help)
	HELP:
		This script will download the latest FDA database
		and adjust the settings so that updates are stored,
		then applied after the databse is fully populated.
		Usage: ./populateDB.sh [OPTION] (use option -h for help)
			-s: output settings only
			-h: print help
