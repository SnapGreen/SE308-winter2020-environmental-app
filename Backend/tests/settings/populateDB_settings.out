settings check:
	SETTINGS: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/tests/settings.txt
	SERVER_POPULATED: 
	USAGE:
		Usage: ./populateDB.sh [OPTION] (use option -h for help)
	HELP:
		This script will download the latest FDA database
		and adjust the settings so that updates are stored,
		then applied after the databse is fully populated.
		Usage: ./populateDB.sh [OPTION] (use option -h for help)
			-s: output settings only
			-h: print help
